Bill3.java

```diff
@@ -2,7 +2,12 @@
 public class Bill3 {
 	public static void main(String[] args) {
 		// To get you started, here is the first line in the program:
-	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+	    String name1 = (args[0]);
+		String name2= (args[1]);
```

Comment:
Throughout the assignment, the white-spacing is not correct.
For example, here the proper white-spacing is "int x = Integer.parseInt(args[0]);"
Notice that there is a space before and after the equal sign.

This applies to variables, conditions, loops etc.

Please make sure to read the [Java Code Style Guidelines](https://drive.google.com/open?id=1Wwoi_ALLpl0xiuyLCTvfsmhTgtMSyuZB&usp=drive_fs), and apply these rules in the next assignment!